myWeek/wsgi.py

```diff
@@ -8,6 +8,8 @@
 """
 
 import os
+from dotenv import load_dotenv
+load_dotenv()
 
 from django.core.wsgi import get_wsgi_application
 
```

Comment:
Did you create this file? // did you put this on a server somewhere? Just curious if this is in preparation to make this live

Reply:
Sorry, that's a library that can be installed with `pip (or pipenv) install python-dotenv`. It's to make sure the .env file loads. It's unnecessary up to this point but when/if we add things like go live and want to conceal the secret key, use an S3 bucket, hide some APIs or other sensitive info (like email credentials when the invitation function is done), dotenv will be necessary.